broker-daemon/utils/grpc-gateway.js

```diff
@@ -0,0 +1,188 @@
+const requiredGrpc = require('grpc')
```

Comment:
Make sure to add a comment to the top of this file of where we got this code and provide an author shoutout. @martineehrlich

Reply:
done!